EventFiltering/PWGCF/CFFilterAll.cxx

```diff
@@ -1309,6 +1435,14 @@ struct CFFilter {
     return isSelected;
   }
 
+  double translatePhi(double phi)
+  {
+    if (phi < 0) {
+      phi += 2 * M_PI; // Add 2π to make it positive
+    }
+    return phi;
+  }
```

Comment:
There is [`RecoDecay::constrainAngle`](https://github.com/AliceO2Group/O2Physics/blob/757752c4aa7755285d8e4f0bec1ebb8bb6824a2c/Common/Core/RecoDecay.h#L215) for this